keps/sig-auth/4193-bound-service-account-token-improvements/README.md

```diff
@@ -164,10 +167,10 @@ The node assertion can be checked to ensure the host identity matches the node a
 Bob is an administrator of a cluster and has noticed some strange request patterns from an unknown service account.
 
 Bob would like to understand who initially issued/authorised this token to be issued. To do so, Bob looks up the JTI
-of the token making the suspicious requests by looking inside the audit log entries for these suspect requests.
+of the token making the suspicious requests by looking inside the audit log entries at user's ExtraInfo for these suspect requests.
 
 This JTI is then used for a further audit log lookup - namely, looking for the TokenRequest `create` call which contains
-the audit annotation with key `authentication.kubernetes.io/token-identifier` and the value set to that of the suspect token.
+the audit annotation with key `authentication.kubernetes.io/issued-credential-id` and the value set to that of the suspect token.
 
 This allows Bob to determine precisely who made the original request for this token, and (depending on the 'chain'
 above this token), allows Bob to recursively perform this lookup to find all involved parties that led to this token
```

Comment:
I've gone through and done all these, though for (3), this whole area (bound service account token issuance) doesn't have any e2e or conformance tests as it is - it's all being tested via integration tests at the moment.

Does that mean I should do the same here (which is already complete), or is e2e/conformance testing for all changes a new requirement added since we added bound tokens in ~1.21-ish?

Reply:
> I've gone through and done all these, though for (3), this whole area (bound service account token issuance) doesn't have any e2e or conformance tests as it is - it's all being tested via integration tests at the moment.

Integration seems sufficient, but please make sure to link it in the appropriate section. About e2e I'll leave it up to you and auth approvers.